PBM-1383 - highlight more word `every` and mention election process (#204)
percona/pbm-docs: docs/details/architecture.md

Percona Backup for MongoDB consists of the following components:

* [`pbm-agent`](../reference/glossary.md#pbm-agent) is a process running on **every** `mongod` node that is not an arbiter node within the cluster or within a replica set that performs backup and restore operations.

* [`pbm` CLI](../reference/glossary.md#pbm-cli) is a command-line utility that instructs `pbm-agents` to perform an operation.

* [PBM Control collections](../reference/glossary.md#pbm-control-collections) are special collections in MongoDB that store the configuration data and backup states. Both `pbm` CLI and `pbm-agent` use PBM Control collections to check backup status in MongoDB and communicate with each other.

* Remote backup storage is where Percona Backup for MongoDB saves backups. It can be either [an S3 compatible storage](../reference/glossary.md#s3-compatible-storage) or a filesystem-type storage.

> ??? info "Nomination and election process within PBM"

Reviewer: I suggest we add this info panel to the [pbm-agent](https://docs.percona.com/percona-backup-mongodb/details/pbm-agent.html) page instead, since it already has a brief explanation of the election process, and then just add the info that a running agent in the primary node is paramount to that process.

Author: done

>      If the `pbm-agent` running on the primary node of the replica set or in the primary node of the config replica set fails, the backup will not start as these agents are responsible for internal nomination process. It is therefore required ensuring that all the `pbm-agent` processes are up and running.

Reviewer 2: ```suggestion
     If the `pbm-agent` running on the primary node of the replica set or on the primary node of the config replica set fails, the backup will not start as these agents are responsible for internal nomination process. You must therefore ensure that all the `pbm-agent` processes are up and running.
```
